config.md

```diff
@@ -49,6 +49,14 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 }
 ```
 
+### Example (Windows)
+
+```json
+"root": {
+    "path": "\\\\?\\Volume{ec84d99e-3f02-11e7-ac6c-00155d7682cf}"
```

Comment:
Actually, it's better that we follow MSDN from the spec. I will update the runtime to ensure the trailing backslash is removed if present before passing onto HCS. Updated here accordingly.